package.json

```diff
@@ -1,30 +1,14 @@
 {
-  "name": "my-app",
-  "version": "0.1.0",
+  "name": "react-template",
+  "version": "2.0.0",
   "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.5.2",
-    "@types/node": "^16.18.3",
-    "@types/react": "^18.0.24",
-    "@types/react-dom": "^18.0.8",
-    "env-cmd": "^10.1.0",
-    "eslint-plugin-eslint-comments": "^3.2.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
-    "typescript": "^4.8.4",
-    "web-vitals": "^2.1.4"
-  },
   "scripts": {
-    "start": "env-cmd -f .env.development react-scripts start",
-    "build_preprod": "env-cmd -f .env.preproduction react-scripts build",
-    "build_prod": "env-cmd -f .env.production react-scripts build",
-    "test": "env-cmd -f .env.development react-scripts test",
+    "start": "vite --mode development",
+    "build_preprod": "tsc && vite build --mode preproduction",
+    "build_prod": "tsc && vite build --mode production",
+    "test": "jest --module=es2020",
     "lint": "eslint \"./src/**/*.ts\" \"./src/**/*.tsx\"",
-    "coverage": "npm test -- --coverage --reporter=html --watchAll",
+    "coverage": "npm test --coverage --reporter=html --watchAll",
```

Comment:
You need the `--` between test and coverage here as it was before -> `npm test -- --coverage ...`.
Otherwise, arguments are given to npm instead of the underlying command